common/web/types/src/kmx/kmx-plus.ts

```diff
@@ -513,6 +514,18 @@ export class List extends Section {
     }
     return this.allocList(strs, s.split(' ').map(unescapeString));
   }
+  /** perform string variable, marker, and unescaping */
+  allocListFromSubstitutedSpaces(s: string, sections: DependencySections): ListItem {
+    if(s === undefined || s === null) {
+      s = '';
+    }
```

Comment:
```suggestion
    s = s ?? '';
```

Seems shorter :grin:

Ref https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing